sql/src/main/java/org/apache/druid/sql/calcite/rel/DruidQuery.java

```diff
@@ -1307,7 +1308,7 @@ private ScanQuery toScanQuery()
         );
         return null;
       }
-    }
+    }*/
```

Comment:
We should just remove this commented code.

Reply:
I have removed it